src/components/LoginForm/index.jsx

```diff
@@ -26,6 +26,8 @@ class LoginForm extends React.Component {
 
   componentDidUpdate(prevProps) {
     if (this.props.isAuthenticated && !prevProps.isAuthenticated) {
+      // TODO! add logic to get the username of currently loggedin user
+      this.props.fetchUserProfile('staff');
```

Comment:
Until the username is in the JWT token and we determine how we're going to fetch the profile image (putting it in the JWT token, or the API request), we should probably not call `fetchUserProfile` here.

@georgebabey Do you think we should ship the actions/reducers/constants associated with `fetchUserProfile` if we're always going to use the default avatar for the time being?

Reply:
Knowing that we'll fast follow when the JWT token ships (and I confirmed with @douglashall the username will be there) I'd just leave this in for now. Maybe update the actions so it returns the default instead of calling the service?

Reply:
@muhammad-ammar @georgebabey Now that the @douglashall's auth change has merged, the username should exist in the `authentication` Redux state.

@muhammad-ammar If you rebase on `master`, are you able to access the username in the `authentication` state? However, with the authentication changes, we no longer have a `LoginPage` component/container so this call would need to moved elsewhere.